packages/app/background-jobs-common/test/processors/TestForeverRescheduledBackgroundJobProcessor.ts

```diff
@@ -21,13 +22,10 @@ export class TestForeverRescheduledBackgroundJobProcessor extends AbstractBackgr
 
   constructor(dependencies: BackgroundJobProcessorDependencies<Data>, redisConfig: RedisConfig) {
     super(dependencies, {
-      queueId: 'TestStalledBackgroundJobProcessor queue',
```

Comment:
This appears to be a leftover from copy-pasting and doesn't impact functionality. Updating it to align with the approach used in other test job processors for consistency.